pkg/ddevapp/mutagen.go

```diff
@@ -820,9 +820,10 @@ func GetDefaultMutagenVolumeSignature(_ *DdevApp) string {
 
 // checkMutagenUploadDirs just tells people if they are using mutagen without upload_dir
 func (app *DdevApp) checkMutagenUploadDirs() {
-	if app.IsMutagenEnabled() && !app.IsUploadDirsDisabled() && len(app.GetUploadDirs()) == 0 {
+	if app.IsMutagenEnabled() && !app.IsUploadDirsWarningDisabled() && len(app.GetUploadDirs()) == 0 {
 		util.Warning("You have Mutagen enabled and your '%s' project type doesn't have `upload_dirs` set.", app.Type)
 		util.Warning("For faster startup and less disk usage, set upload_dirs to where your user-generated files are stored.")
-		util.Warning("If this is intended you can disable this warning by running `ddev config --upload-dirs=false`.")
+		// TODO: Implement --no-upload-dirs-warning in new PR
+		util.Warning("If this is intended you can disable this warning with `ddev config --no-upload-dirs-warning`.")
```

Comment:
```suggestion
		//util.Warning("If this is intended you can disable this warning with `ddev config --no-upload-dirs-warning`.")
```
does not make much sense to show this info even it doesn't work at the moment

Reply:
I'm going to follow it immediately with the actual replacement, so it's ok.